v_4_0_0/filemanager.go

```diff
@@ -0,0 +1,43 @@
+package v_4_0_0
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+// Files is map[string]string for files that we fetched from disk and then filled with data.
+type Files map[string]string
+
+func RenderFiles(filesdir string, ctx interface{}) (Files, error) {
+	files := Files{}
+
+	err := filepath.Walk(filesdir, func(path string, f os.FileInfo, err error) error {
+		if f.Mode().IsRegular() {
+			tmpl, err := template.ParseFiles(path)
+			if err != nil {
+				return microerror.Maskf(err, "failed to parse file #%q", path)
+			}
+			var data bytes.Buffer
+			tmpl.Execute(&data, ctx)
+
+			relativePath, err := filepath.Rel(filesdir, path)
+			fmt.Printf("path: %s", relativePath)
```

Comment:
Is this print intentional for final use or possible leftover from debugging?

Reply:
leftover